linodecli/overrides.py

```diff
@@ -37,3 +41,67 @@ def handle_domains_zone_file(operation, output_handler, json_data) -> bool:
     """
     print("\n".join(json_data["zone_file"]))
     return False
+
+
+@output_override("linodes", "types", OutputMode.table)
+def handle_types_region_prices_list(
+    operation, output_handler, json_data
+) -> bool:
```

Comment:
We may want to move this code out into a separate function so we can partially re-use it for ASCII and markdown outputs :+1: